henry/commands/vacuum.py

```diff
@@ -15,7 +15,7 @@ def run(cls, user_input: fetcher.Input):
         vacuum.output(data=cast(fetcher.TResult, result))
 
     @spinner.Spinner()
-    def models(self, *, project: Optional[str] = None, model: str) -> fetcher.TResult:
+    def models(self, *, project: Optional[str] = None, model: Optional[str]) -> fetcher.TResult:
```

Comment:
Making `model` optional sounds correct, but please walk me through your reasoning to jog my memory.

Reply:
My mistake - the `fetcher.TResult` class declares model as `Optional[str]` but that's because it's shared between the various commands, and in some places it is optional. My linter caught this as an incompatibility but we should ignore as `vacuum.models` really does require a string to be passed